app/keepers/keepers.go

```diff
@@ -498,12 +516,13 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 	appKeepers.IBCKeeper.SetRouter(ibcRouter)
 
 	// register the proposal types
-	govRouter := govtypes.NewRouter()
-	govRouter.AddRoute(govtypes.RouterKey, govtypes.ProposalHandler).
+	govRouter := govtypesv1.NewRouter()
+	govRouter.AddRoute(govtypes.RouterKey, govtypesv1.ProposalHandler).
 		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(*appKeepers.ParamsKeeper)).
-		AddRoute(distrtypes.RouterKey, distribution.NewCommunityPoolSpendProposalHandler(*appKeepers.DistrKeeper)).
+		// UNFORKINGTODO C: I think it got depricated in favor of MsgCommunityPoolSpend, but need to make sure this is okay to remove.
+		//AddRoute(distrtypes.RouterKey, distribution.NewCommunityPoolSpendProposalHandler(*appKeepers.DistrKeeper)).
```

Comment:
I think we can keep both for legacy compat for now

Reply:
No the problem is, `NewCommunityPoolSpendProposalHandler` completely doesn't exist anymore, so we can't keep it. Just noting this here in case we need to do something in place of removing this that I am not aware of.

Reply:
Yeah I am pretty sure we just submit a new prop type to do a community pool spend, since this doesn't exist anymore, going to remove